demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -518,7 +543,7 @@ def copy_packs_content_to_packs_bundle(self, packs):
                                 continue
                             unifier = Unifier(package_dir, dir_name, dest_dir)
 
-                            if parse_version(unifier.yml_data.get('fromversion', '0.0.0')) >= parse_version('6.0.0'):
+                            if parse_version(unifier.yml_data.get('fromversion', '0.0.0')) <= parse_version('6.0.0'):
```

Comment:
shouldn't the filter for packs be related to `toversion` instead of `fromversion`?

Reply:
Correct me if I am wrong but the idea is that content entities added in 6.0.0 should no be added to content_new.zip as the are "self-releasing" hence it should be that they are :`fromversion: 6.0.0`

Reply:
but what about content entities that are already exists?
for example, Integration with `fromversion: 5.0.0` should be included in the packs.

Reply:
done